src/install.rdf

```diff
@@ -6,8 +6,7 @@
         RDF:about="urn:mozilla:install-manifest"
         em:id="zotero-ocr@bib.uni-mannheim.de"
         em:name="Zotero OCR"
-        em:version="0.6.0"
-        em:type="2"
+        em:version="1.0.0"
```

Comment:
Below here in the install.rdf you need to increase the minVersion to 6.0 twice, as the json-style update works as far as we know from then on only.

Reply:
Thanks for the hint. That's done now.